StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -297,10 +298,13 @@ Scene* LeosTestScene(SceneManager* sm)
     melc = entity->AddComponent<component::MeleeComponent>();
     ranc = entity->AddComponent<component::RangeComponent>(sm, scene, sphereModel, 0.3, 1, 20);
     uc = entity->AddComponent<component::UpgradeComponent>();
+    ac = entity->AddComponent<component::AnimationComponent>();
 
 
     mc->SetModel(playerModel);
-    mc->SetDrawFlag(FLAG_DRAW::DRAW_ANIMATED | FLAG_DRAW::GIVE_SHADOW);
+    mc->SetDrawFlag(FLAG_DRAW::DRAW_ANIMATED | FLAG_DRAW::GIVE_SHADOW | FLAG_DRAW::NO_DEPTH);
```

Comment:
Why should the player have no depth flag?

Reply:
Because the animations are applied in a separate depth pass. This means the regular depth pass should not run, else we basically run depth twice for the animated models.